src/__Private/LintRunConfig.hack

```diff
@@ -84,7 +84,7 @@ final class LintRunConfig {
     HHAST\ConsistentLineEndingsLinter::class,
   ];
 
-  const vec<classname<SingleRuleLinter>> NON_DEFAULT_LINTERS = vec[
+  const vec<classname<Linter>> NON_DEFAULT_LINTERS = vec[
```

Comment:
Doesn't need changing here, but when HHClientLinter is added, this mechanism won't be sufficient